bentoml/_internal/bento/bento.py

```diff
@@ -277,8 +264,23 @@ def from_yaml_file(cls, stream: t.IO[t.Any]):
             logger.error(exc)
             raise
 
-        bento_info = cls(**yaml_content)
-        bento_info.validate()
+        yaml_content["tag"] = Tag(yaml_content["name"], yaml_content["version"])
+        del yaml_content["name"]
```

Comment:
Why do we have to do this?

Reply:
I'm passing the yaml directly as kwargs to init.